test/unit/plugins/rest_spec.js

```diff
@@ -797,18 +831,102 @@ describe("The Rest plugin", function () {
 				expect (failed, "concurrency").to.be.true;
 			});
 		});
+
+		describe("setting the status of an instance to a state not handled by update", function () {
+
+			var instanceID;
+			var responseCode;
+			var originalInstance;
+
+			before(function () {
+
+				return instances.createInstance()
+				.then(function (instance) {
+					instanceID = instance.id;
+					return instances.getInstance({ id : instance.id });
+				})
+				.then(function (instance) {
+					originalInstance = {
+						id       : instance.id,
+						ami      : instance.ami,
+						type     : instance.type,
+						uri      : instance.uri,
+						state    : instance.state,
+						revision : instance.revision
+					};
+
+					var request = new Request("PUT", "/v1/instances/" + instance.id).mime("application/json")
+					.payload({
+						ami      : instance.ami,
+						type     : instance.type,
+						uri      : instance.uri,
+						state    : "failed",
+						revision : instance.revision
+					});
+					return request.inject(server);
+				})
+				.then(function (response) {
+					responseCode = response.statusCode;
+				});
+			});
+
+			it("throws an error", function () {
+				return instances.getInstance({ id : instanceID })
```

Comment:
Is it to verify that the instance is not modified in the event of an error? If that is the case, it would probably be clearer (and more self-descriptive) to separate this out into it's own assertion (`it` block).